instaloader/__main__.py

```diff
@@ -111,6 +112,10 @@ def import_session(browser, instaloader, cookiefile):
         if not username:
             raise SystemExit(f"Not logged in. Are you logged in successfully in {browser}?")
         instaloader.context.username = username
+        print(f"{username} has been successfully logged in.")
+        next_step_text = (f"Next: Run instaloader --login={username} as it is required to download high quality media "
+            "and to make full use of instaloader's features.")
```

Comment:
We always write capitalized `Instaloader`, with a capitalized `I`, unless we refer to the CLI command or the python module.

Also I'd indent the second line further, so that it has the same indentation as the opening parenthesis in the first line.

Reply:
Done.